build_tools/travis/before_install.sh

```diff
@@ -35,43 +36,41 @@ if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
 else
     # Install some custom requirements on Linux
     # g++4.8.1
-    if [ "$CXX" == "g++" ]; then sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test; fi
+    sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test; 
+    sudo apt-get install -qq g++-4.8;
+    export CXX="g++-4.8"; 
```

Comment:
**anyway I'll reformat that file soon,** but as on travis the ubuntu version is old (trusty is 14.04), the libopenblas is bugged (not all the files as openblas_config.h), so I need the 
sudo sed -i -e 's/trusty/vivid/g' /etc/apt/sources.list
and when it was built on travis, I had a clang build failed whereas the g++ was ok, and this because of libopenblas .... there should not be any reason for that, but adding the g++ ppa, solved this...